sys/include/evtimer.h

```diff
@@ -105,6 +105,14 @@ void evtimer_del(evtimer_t *evtimer, evtimer_event_t *event);
  */
 void evtimer_print(const evtimer_t *evtimer);
 
+/**
+ * @brief   Return the current system time in msec
+ */
+static inline uint32_t evtimer_now_msec(void)
+{
+    return (xtimer_now_usec64() / US_PER_MS) & UINT32_MAX;
```

Comment:
Why `& UINT32_MAX` here? This decreases the possible value space for `_now_min()` below.

Reply:
Hmm, I just extracted that line from all the replacements I did.

To be honest, I do not understand the need for the manual bit truncation using `& UINT32_MAX` when casting uint64_t to uint32_t. Can you maybe give me a hint?

Reply:
> To be honest, I do not understand the need for the manual bit truncation using & UINT32_MAX when casting uint64_t to uint32_t. Can you maybe give me a hint?

It is not necessarily needed, it just makes the truncation more visible and marks it as intentionally.

> Hmm, I just extracted that line from all the replacements I did.

For most of your replacements you are, however, `_now_min()` uses the full value space of uint64_t, converts that to minutes and only then truncates it. The value space in current master of possible minute values is [0, 2503056545], while with your version it becomes [0, 71]. Given that the protocols using these minute values are designed to work in terms of weeks, this is significant.